### ISSUE TITLE
Exception not passed in payload when directly logging (partially commented out script)

### ISSUE THREAD

**Issue by and-megan** [rollbar/rollbar.js#333]
Commented out the code that saves things under 'window' and code that logs in the console.

Using the following syntax to log exceptions directly: 
```
const reporterInstance = new Rollbar({...})
try {
  // something dangerous
} catch (e) {
  reporterInstance.error(e, extraData);
}
```

The actual exceptions are absent in the payload, but data is being passed in body.message.extra.

**Comment by rokob** [comment]
What does this mean:
> Commented out the code that saves things under 'window' and code that logs in the console.

**Comment by and-megan** [comment]
These parts are commented out from the snippet.

/*&&(window._rollbarStartTime=(new Date).getTime())*/

/*window.rollbar=n,window._rollbarDidLoad=!0*/

/*"use strict";t.console||(t.console={});for(var e,r,n=t.console,o=function(){},i=["memory"],a="assert,clear,count,debug,dir,dirxml,error,exception,group,groupCollapsed,groupEnd,info,log,markTimeline,profile,profiles,profileEnd,show,table,time,timeEnd,timeline,timelineEnd,timeStamp,trace,warn".split(",");e=i.pop();)n[e]||(n[e]={});for(;r=a.pop();)n[r]||(n[r]=o)*/

**Comment by rokob** [comment]
If you comment out some subset of the minified snippet, I can't make any real guarantees about what will happen. It is not a bug that things break from commenting out certain parts of the snippet. What do they want to be happening? We can maybe work on a different build that does some things,  for instance they can load the .noconflict.umd.min.js directly instead of using the snippet to probably get the same effect they are going for but that also works.

**Comment by ewino** [comment]
Hi rokob,

The reason those parts were commented out is that they change the global window object. This code using rollbar is a third party script that tries to have as little a footprint as possible (e.g. not polluting the global namespace).

We went through the instructions here https://github.com/rollbar/rollbar.js/tree/master/examples/webpack and downloaded the specified file. Since _that_ file also changes the window object (e.g sets window.rollbar) we commented out parts of it. 

The reason we didn't use the noconflict version is that it too (at least according to the docs) changes the window by adding _rollbarStartTime. 

Rollbar is definitely awesome. We love it. Is there any way we could use it without it looking for global configs or saving its constructor and start-time to the window object? Maybe some sort of configurable option in the noconflict version to do without the start time? or even better, save the start time in a private scope, unaccessible from the window object?
